src/devices/x11window_xlib.c

```diff
@@ -199,6 +222,47 @@ static rgb_fmt_t x11_get_rgb_format(Display* display)
     return format;
 }
 
+static void x11_handle_keypress(fb_window_t* win, hid_key_t hid_key)
+{
+    Display* dsp = win->data->display;
+
+    if (win->data->grab_modkeys == 0
+            && win->data->pressed_modkeys & (KEY_MASK_LEFTCTRL | KEY_MASK_RIGHTCTRL)
+            && win->data->pressed_modkeys & (KEY_MASK_LEFTALT | KEY_MASK_RIGHTALT)
+            && hid_key == HID_KEY_G) {
+        win->data->grab_modkeys = win->data->pressed_modkeys;
+
+        XGrabKeyboard(dsp, win->data->window, True, GrabModeAsync,
+                GrabModeAsync, CurrentTime);
+        XGrabPointer(dsp, win->data->window, True,
+                ButtonPressMask | ButtonReleaseMask | PointerMotionMask,
+                GrabModeAsync, GrabModeAsync, win->data->window, None, CurrentTime);
+
+        char buf[128], *p = buf;
+        p = stpcpy(p, "RVVM - Press ");
```

Comment:
I do have suspicion this will not be supported on some targets... Correct if I'm wrong